### DISCUSSION TITLE
which is the better way for instal clamav and rspamd and config with Postal?

### DISCUSSION THREAD

**Discussion by mredodos** [postalserver/postal#2657] [Help with using Postal] [answered]
Hi some one can help me to figure out the better way to install clamav and rspamd with postal?

**Comment by willpower232** [comment]
when you say "the better way", what way do you have at the minute?

**Reply by mredodos** [reply]
i dont understand if exist a docker image with settings(it seems yes) for postal or i need just install via docker clamav and rspamd and after postal make automatically all the setting, i dont understand the correct way

im confused and i dont have found a documentation

**Reply by willpower232** [reply, marked as answer]
Ah right, well you can run them however you want to. Postal defaults to host networking so it should be able to reach them however you run them.

All you need to do is configure some or all of these options in the config file then that should do the trick.

https://github.com/postalserver/postal/blob/2f62baa238fc1102706ee4acf079b7a876b05283/config/postal.defaults.yml#L110-L126

**Reply by mredodos** [reply]
so can I also install them on other docker images with their own address or in external servers? good.

 i need to do something else special configuration or other staff inside the postal or clamaav for example??

**Reply by willpower232** [reply]
> so can I also install them on other docker images with their own address or in external servers? good.

yep

> i need to do something else special configuration or other staff inside the postal or clamaav for example??

not to my knowledge but its been a while since I set up Postal from scratch

**Reply by mredodos** [reply]
thx a lot
